src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -90,244 +90,256 @@ public KubernetesLauncher() {
 
     @Override
     public boolean isLaunchSupported() {
-        return !launched;
+        // Don't synchronize here to avoid deadlocks
+        return true;
```

Comment:
This is incorrect. You are missing the purpose of this method or why there is a persisted field `launched`.